rosapi/src/rosapi/proxy.py

```diff
@@ -45,16 +45,20 @@
 
 from rosapi.msg import TypeDef
 
+from glob_helper import filter_globs, any_match
```

Comment:
Use `from .glob_helper import filter_globs, any_match` to import from local module.